audiofield/fields.py

```diff
@@ -135,9 +135,14 @@ def _convert_audio(self, filename, instance=None, ext=None):
         filename_temp = filename[:-4] + '_temp'
 
         # Find the number of channels
-        if os.path.isfile(filename):
-            command = "soxi -c %s" % filename
-            response = subprocess.Popen(command.split(' '), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if os.path.isfile(filename): 
+            command = settings.SOX_CLI_COMMAND + "%s" % filename
```

Comment:
check if SOX_CLI_COMMAND exist and it's defined if not fallback on a default value